### DISCUSSION TITLE
各种不同下载格式有何不同？

### DISCUSSION THREAD

**Discussion by treefive3000** [AlistGo/alist#4187] [Q&A]
您好！请问同一版本不同下载格式有何不同，详看截图。
谢谢
![image](https://user-images.githubusercontent.com/32722117/232695510-c777b25c-b4c2-4561-8a31-d0619c66cdfa.png)

**Comment by anwen-anyi** [comment]
386自己百度

arm是架构

amd-upx 是压缩了的 电脑可能会误报病毒 误删 实际上没有问题的 节省资源12M下载 实际40M的

amd是没压缩的 介意upx压缩误报的用这个
